pebblo/app/service/local_ui_service.py

```diff
@@ -367,3 +380,57 @@ def get_vector_dbs(chains):
         for data in chains:
             vector_dbs.extend([db["name"] for db in data["vectorDbs"]])
         return list(set(vector_dbs))
+
+    @staticmethod
+    def get_all_documents(retrieval_data):
+        """This function returns documents per app"""
```

Comment:
Please update doc-string mentioning it returns sorted list of documents based on number of times it appeared in the retrievals.